src/balancereader/src/test/java/anthos/samples/bankofanthos/balancereader/BalanceReaderControllerTest.java

```diff
@@ -63,7 +74,28 @@
     @BeforeEach
     void setUp() {
         initMocks(this);
-        balanceReaderController = new BalanceReaderController(ledgerReader, verifier, cache, LOCAL_ROUTING_NUM, VERSION);
+        StackdriverMeterRegistry meterRegistry = new StackdriverMeterRegistry(new StackdriverConfig() {
+            @Override
+            public boolean enabled() {
+                return false;
+            }
+
+            @Override
+            public String projectId() {
+                return "test";
+            }
+
+            @Override
+            @Nullable
+            public String get(String key) {
+                return null;
+            }
+        }, clock);
+
+        when(cache.stats()).thenReturn(new CacheStats(0,
+            0,0,0,0,0));
```

Comment:
just curious, what do all these 0s represent? are they required for initialization? is there a default value for the CacheStats object?

Reply:
There isn't a default value for the Cache Stats object and this is required since in the initialization of the balanceReader, the guavacachemetrics calls cache.stats().hitcount(), cache.stats().misscount(),etc and since the cache is a Mock the cache.stats() originally returned null which caused a nullPointerException.

Reply:
Turned the cache stats object into a mock to clean things up!